### ISSUE TITLE
Setting up Keras Bert for Reading Comprehension 

### ISSUE THREAD

**Issue by Santosh-Gupta** [CyberZHG/keras-bert#47]
I am looking at section 4.2 of the BERT paper or how to set up BERT for reading comprehension. It looks like a module needs to be added to the end of BERT, S and E are new parameters, and a log softmax loss is calculated for the start and end positions. 

This extension is included in the original tensorflow BERT in the 'run_squad.py' script in the repository. 

Does does an extension exist for BERT Keras?
